### DISCUSSION TITLE
Does mini OS have systemd boot?and a new linux kernel

### DISCUSSION THREAD

**Discussion by ghost** [minios-linux/minios-live#29] [Q&A]
I need  to check if it meet the requirement to run on my chromebook,so does it have systemd boot and a linux kernel version of Greater that 6.4?

**Comment by FershoUno** [comment]
Hello, MiniOS uses Systemd, the kernel version it uses is 6.1.
You can try MiniOS without installing it and see if it recognizes all your hardware.

**Comment by crim50n** [comment]
> does it have systemd boot and a linux kernel version of Greater that 6.4?

You can try to build a module with the required kernel yourself, out of the box MiniOS does not support Secure Boot and ships with kernel 6.1. If I have time, I will post here instructions on how to build a kernel module with Secure Boot support.
